packages/pluggableWidgets/popup-menu-web/src/__tests__/PopupMenu.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { shallow, ShallowWrapper } from "enzyme";
+import { createElement } from "react";
+import { actionValue } from "../../../../tools/util-widgets/dist";
+import { BasicItemsType, CustomItemsType, PopupMenuContainerProps } from "../../typings/PopupMenuProps";
+import PopupMenu from "../PopupMenu";
+
+jest.useFakeTimers();
+
+describe("Popup menu", () => {
+    const createPopupMenu = (props: PopupMenuContainerProps): ShallowWrapper<PopupMenuContainerProps, any> =>
```

Comment:
Is this `any` necessary? Looks like state is default to `{}` in Generic.

Reply:
fixed

Reply:
I see you changed to `{}`, which is fine, although the generic `ShallowWrapper` will default state to this value if you don't pass it. Just a FYI.